modules/aws/dynamodb.go

```diff
@@ -0,0 +1,89 @@
+package aws
+
+import (
+	"testing"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/dynamodb"
+	"github.com/gruntwork-io/terratest/modules/logger"
+)
+
+func CreateDynamodbTable(t *testing.T, region string, name string) {
+	err := CreateDynamodbTableE(t, region, name)
+	if err != nil {
+		t.Fatal(err)
+	}
+}
+
+func CreateDynamodbTableE(t *testing.T, region string, name string) error {
+	logger.Logf(t, "Creating DynamoDb table '%s' in %s", name, region)
+
+	dynamodbClient, err := NewDynamodbClientE(t, region)
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	params := &dynamodb.CreateTableInput{
+		AttributeDefinitions: []*dynamodb.AttributeDefinition{
+			{
+				AttributeName: aws.String("Terratest"),
+				AttributeType: aws.String("S"),
+			},
+		},
+		KeySchema: []*dynamodb.KeySchemaElement{
+			{
+				AttributeName: aws.String("Terratest"),
+				KeyType: aws.String("HASH"),
+			},
+		},
+		ProvisionedThroughput: &dynamodb.ProvisionedThroughput{
+			ReadCapacityUnits:  aws.Int64(1),
+			WriteCapacityUnits: aws.Int64(1),
+		},
+		TableName: aws.String(name),
+	}
+
+	_, err = dynamodbClient.CreateTable(params)
```

Comment:
The DynamoDB table API is asynchronous and after this returns, the table doesn't actually exist for another 5-30 seconds. You need to add a wait as shown here or many uses of this API will end up hitting errors when they try to use that table too early: https://github.com/gruntwork-io/terragrunt/blob/master/dynamodb/dynamo_lock_table.go#L170